### PR TITLE
Add bank account number validation for Citibank

### PR DESCRIPTION
**Pull request by jpaulohe4rt**
## Description 
Add bank account number validation for Citibank based on this reference:
http://hostmarcasite.com.br/fluir/vendor/eduardokum/laravel-boleto/manuais/Regras%20Validacao%20Conta%20Corrente%20VI_EPS.pdf

### CODE REVIEW

File: src/main/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAccountCitibankValidator.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    /**
+     * Checks whether the specified string represents a valid bank account number of Banco Citibank
+     *
+     * @param bankAccount The sequence to verify.
+     * @return {@code true} if the sequence is a valid bank account number; {@code false} otherwise.
+     */
+    public boolean valid(String bankAccount) {
+        if (StringUtils.isBlank(bankAccount)
+                || (StringUtils.length(bankAccount) > 11)
+                || !bankAccount.matches("\\d+")
```
Comment (alfoltran) [inline src/main/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAccountCitibankValidator.java]: Wouldn't it be at least two digits?

**Review by alfoltran — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAccountCitibankValidator.java (hunk 1 above)
Comment (jpaulohe4rt) [reply src/main/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAccountCitibankValidator.java]: For sure. I will fix this.

Comment (alfoltran) [inline src/main/java/com/github/darkroomdevs/checkdigit/validator/brazil/BankAccountCitibankValidator.java]:
```suggestion
                || !bankAccount.matches("\\d{2,}")
```

**Review by alfoltran — CHANGES_REQUESTED** [review]

**Review by alfoltran — APPROVED** [review]
:ok:

**Pull request merged**